interfaces/api/v1/workbench/monitor.py

```
@@ -42,6 +43,76 @@ class ForeshadowStatsResponse(BaseModel):
 
 
+def _is_default_tension(value: Any) -> bool:
+    """判断张力值是否仍是默认中性值。"""
+    try:
+        return abs(float(value) - 50.0) < 1e-6
+    except (TypeError, ValueError):
+        return False
+
+
+def _needs_tension_backfill(chapter: Any) -> bool:
+    """正文已存在，但张力四字段仍全为默认值时触发懒回填。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)